code/stable/sglpend/src/java/SglPend/Control.java

```diff
@@ -0,0 +1,38 @@
+package SglPend;
+
+/** \file Control.java
+    \author Olu Owojaiye
+    \brief Controls the flow of the program
+    \note Generated by Drasil v0.1-alpha
+*/
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Control {
+    
+    /** \brief Controls the flow of the program
+        \param args List of command-line arguments
+    */
+    public static void main(String[] args) throws FileNotFoundException, IOException {
+        String filename = args[0];
+        double L_rod;
+        double m;
+        double α;
+        double θ_p;
+        double θ_i;
+        double p_x^i;
+        double p_y^i;
+        Object[] outputs = InputParameters.get_input(filename);
```

Comment:
`outputs = Input...`

Reply:
Hey, Drasil does *exactly* what it was asked to do! Fancy that!

Probably `pendDisplacementAngle` should not be an input.